test/unit/protocols/test_rb.py

```diff
@@ -163,9 +163,9 @@ def test_combined_design_access(self):
 
     def test_serialization(self):
 
-        self.irb_design.write('../../test_packages/temp_test_files/test_InterleavedRBDesign_serialization')
+        self.irb_design.write('../../test/test_packages/temp_test_files/test_InterleavedRBDesign_serialization')
```

Comment:
I'm confused about why this change is needed. Aren't we in `test/unit/protocols`, so `../../test_packages` is the right way to get to those files?

Reply:
I thought so originally myself, but I noticed when running unit tests locally last night that a new `test_packages` directory was being created at the base level (i.e. at the same level as `test` and `pygsti`) and populated with these files. In retrospect I suppose I could have equivalently removed one of the sets of '..'. Not sure why this is the case though, tbh, maybe something to do with how pytest sets the cwd?

Reply:
It is also possible this was related to which directory I was launching pytest from, so if you have a venv handy to run these tests on and have a different experience that'd be good to know.